platform/ios/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
+## 3.6.4
+
+* Fixed an issue where offline raster tiles does not use correct zoom level. ([#10012](https://github.com/mapbox/mapbox-gl-native/pull/10012))
```

Comment:
From looking at #10011 (the `master` parent of #10012), it seems like maybe this would be a clearer description of the change:

> Fixed an issue where stale (but still valid) map data could be ignored in offline mode.